.github/workflows/dev-email-worker.yml

```diff
@@ -11,25 +11,89 @@ on:
     branches:
       - main
 
-jobs:
+
+jobs:  
   unit-test:
     uses: SierraSystems/reusable-workflows/.github/workflows/java-unit-tests.yml@main
     with:
       working_directory: "src"
       profile: dps-email-worker
     secrets:
       nexus_url: ${{ secrets.NEXUS_URL }}
-
+   
   app-version:
-    uses: SierraSystems/reusable-workflows/.github/workflows/java-maven-app-version.yml@main
-    with:
-      working_directory: "src/dps-email-worker"
+    name: Get the app-version from the POM file
+    runs-on: ubuntu-latest
+    outputs:
+      app-version: ${{ steps.get-version.outputs.app-version }}
+    defaults:
+      run:
+        working-directory: "src/dps-email-worker"
+  
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: "Set up JDK 8"
+        uses: actions/setup-java@v2
+        with:
+          distribution: 'temurin'
+          java-version: 8
+
+      
+      # Runs build steps
+      - name: Maven Package
+        run: mvn -ntp clean package
 
-  build-image:
+      - name: Get Java Project Version
+        id: get-version
+        run: |
+          _version=$(mvn help:evaluate -Dexpression=project.version -q -DforceStdout)
+          echo "::set-output name=app-version::$_version"
```

Comment:
set-output is deprecated.  new syntax is:
echo "{app-version}=$_version" >> $GITHUB_OUTPUT